modules/op-release-notes-1-8.adoc

```diff
@@ -44,8 +44,29 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
 
-// Pipelines
-// *
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even if GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* With this update, as a developer, you can use the *Developer* perspective of the {product-title} web console to run all the `PipelineRun` resources in your `.tekton` directory that match specified events. The console displays the status and reports of the runs. If you are using WebVCS, the reports display comments.
+// (link:https://issues.redhat.com//browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, as an administrator, you can install Tekton Hub with minimal input because it has better default configuration settings. You can start running Tekton Hub and work with your teams to decide what customizations they need.
```

Comment:
this is also a new feature. description is fine 👍🏽

Reply:
Updated. Thanks.